src/main/java/com/tobmistaketracker/detector/VerzikP3MistakeDetector.java

```diff
@@ -92,7 +90,11 @@ public List<TobMistake> detectMistakes(@NonNull TobRaider raider) {
             mistakes.add(TobMistake.VERZIK_P3_PURPLE);
         }
 
-        // TODO: Verzik melee if the animations become unmasked
+        String chancedMeleePlayer = verzikMeleeChancedTracker.getPlayerThatChancedMelee();
+        if (chancedMeleePlayer != null && chancedMeleePlayer.equals(raider.getName())) {
+            verzikMeleeChancedTracker.setPlayerThatChancedMelee(null);
```

Comment:
We don't need this, since we clear it on `afterDetect` anyway. We can remove this line